python/src/trezorlib/tools.py

```diff
@@ -76,9 +78,7 @@ def tx_hash(data: bytes) -> bytes:
 
 
 def hash_160(public_key: bytes) -> bytes:
-    md = hashlib.new("ripemd160")
-    md.update(hashlib.sha256(public_key).digest())
-    return md.digest()
+    return ripemd160(hashlib.sha256(public_key).digest())
```

Comment:
Btw, the OpenSSL upstream reverted this stupid decision and re-introduced RIPEMD160 back to OpenSSL, so we might do it conditionally, but I am not sure it is worth the hassle.

```suggestion
    try:
        md = hashlib.new("ripemd160")
        md.update(hashlib.sha256(public_key).digest())
        return md.digest()
    except ValueError:
        from ._ripemd160 import ripemd160
        return ripemd160(hashlib.sha256(public_key).digest())
```

Reply:
Yeah, not worth it. This isn't really _used_ anywhere, i am in fact considering removing the failing function from trezorlib. I pushed this change to fix some failing tests, but it might be better to move the functionality into the test suite.